src/functions/searchResults/framework/__tests__/handler.spec.ts

```diff
@@ -84,4 +89,24 @@ describe('searchResults handler', () => {
       moqSearchResults.verify(x => x(It.isObjectWith(queryParameter)), Times.once());
     });
   });
+
+  describe('using valid query parameters as LDTM, 8 digit application reference', () => {
+    it('gets the relevant results', async () => {
+      dummyApigwEvent.requestContext.authorizer = {
+        examinerRole: UserRole.LDTM,
+      };
+      dummyApigwEvent.queryStringParameters['startDate'] = queryParameter.startDate;
+      dummyApigwEvent.queryStringParameters['endDate'] = queryParameter.endDate;
+      dummyApigwEvent.queryStringParameters['driverNumber'] = queryParameter.driverNumber;
+      dummyApigwEvent.queryStringParameters['dtcCode'] = queryParameter.dtcCode;
+      dummyApigwEvent.queryStringParameters['staffNumber'] = queryParameter.staffNumber;
+      dummyApigwEvent.queryStringParameters['applicationReference'] = queryParameterWith8DigitAppRef
```

Comment:
For this test it references queryParameterWith8DigitAppRef only once. I think all of them should reference this instead of queryParameter.

Reply:
The thing is, we are still supporting the original 11 digit app ref, all the existing tests use the 11 digit appRef, this new test checks that we also support 8 digits as the search, which is why it has it's own fixture.

Reply:
Ahhhh, I get what you mean, will update it, misunderstood what you said !!!